src/Files.App/Data/Items/AppLanguageItem.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) 2023 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+using System.Globalization;
+
+namespace Files.App.Data.Items
+{
+	public class AppLanguageItem
+	{
+		public string LanguageID { get; set; }
+
+		public string LanguageName { get; set; }
+
+		public AppLanguageItem(string languagID)
```

Comment:
There's a typo, this should be languageID

Reply:
Done